deepchecks/checks/performance/class_performance_imbalance.py

```diff
@@ -34,11 +34,11 @@
 AlternativeScorer = t.Union[str, ScorerFunc]
 
 
-CP = t.TypeVar('CP', bound='ClassPerformanceImbalance')
+CP = t.TypeVar('CP', bound='ClassPerformance')
 
 
-class ClassPerformanceImbalance(SingleDatasetBaseCheck):
-    """Visualize class imbalance by displaying the difference between class score values.
+class ClassPerformance(SingleDatasetBaseCheck):
+    """Visualize class performance compared to other classes.
```

Comment:
mmm still not really clear, maybe write 
Visualize performance per class - aiding detection of class imbalance

But don't put a lot of effort into it because next sprint there is a task merging this with performance report and train-test overfit